### PR TITLE
Add reverse-engineering topic

### PR DESCRIPTION
**Pull request by remiflavien1**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------
### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> 1,970 repositories for this tag so there is a lot of concerned projects. And reverse engineering is a large domain in the field of cybersecurity. 

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/reverse-engineering/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: reverse
```
Comment (kenyonj) [inline topics/reverse-engineering/index.md:2]: This alias seems a little too broad. Thoughts on removing it?
```suggestion
```

File: topics/reverse-engineering/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (kenyonj) [inline topics/reverse-engineering/index.md:1]:
```suggestion
---
```

Comment (kenyonj) [reply topics/reverse-engineering/index.md:1]: there is something going on with the characters on this line

File: topics/reverse-engineering/index.md (hunk 1 above)
Comment (remiflavien1) [reply topics/reverse-engineering/index.md]: Suggestion taken into account. In fact this a little to large. I removed it

File: topics/reverse-engineering/index.md (hunk 2 above)
Comment (remiflavien1) [reply topics/reverse-engineering/index.md]: I'm on wsl. Performed a `dos2unix` on `index.md`. 
No error localy with `script/cibuild`
We will see result of CI pipeline !

Comment (remiflavien1) [reply topics/reverse-engineering/index.md]: All tests passed

Comment (remiflavien1) [reply topics/reverse-engineering/index.md]: All tests passed.

**Comment by kenyonj** [comment]
@remiflavien1 can you merge master in to your branch?

**Comment by remiflavien1** [comment]
> @remiflavien1 can you merge master in to your branch?

It's Done @kenyonj

**Pull request merged**